tests/benchmark.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Benchmark all algorithms on the rosenbrock function."""
+import argparse
+
+from orion.benchmark.assessment import AverageResult
+from orion.benchmark.benchmark_client import get_or_create_benchmark
+from orion.benchmark.task import Branin, RosenBrock
+
+
+def main(argv=None):
+    """Execute the benchmark with cli"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--repetitions",
+        type=int,
+        default=5,
+        help="Number of repetitions for each benchmark.",
+    )
+    parser.add_argument(
+        "--max-trials",
+        type=int,
+        default=100,
+        help="Number of trials for each run",
+    )
+    parser.add_argument(
+        "--output",
+        type=str,
+        default=".",
+        help="Folder where to save figures.",
+    )
+
+    options = parser.parse_args(argv)
+
+    benchmark = get_or_create_benchmark(
+        name="benchmark_ax_optimizer",
+        algorithms=["random"],
```

Comment:
```suggestion
        algorithms=["random", "axoptimizer"],
```
The ax optimizer was missing here.